app/src/main/res/values/styles.xml

```diff
@@ -18,4 +18,22 @@
         <item name="android:backgroundTint">@color/teal_700</item>
         <item name="android:textAllCaps">false</item>
     </style>
+
+    <style name="RecyclerViewMainActivityStyle" parent="TextAppearance.AppCompat.Display1">
+        <item name="android:layout_marginTop">@dimen/recycler_view_main_activity_margin_top</item>
+        <item name="android:layout_marginBottom">@dimen/recycler_view_main_activity_margin_bottom</item>
+    </style>
+
+    <style name="TextViewItemRecyclerViewStyle" parent="TextAppearance.AppCompat.Display1">
+        <item name="android:textStyle">bold</item>
+        <item name="android:fontFamily">sans-serif-condensed</item>
+        <item name="android:textColor">@color/blue</item>
+        <item name="android:textSize">@dimen/text_view_item_recycler_view_size</item>
+        <item name="android:gravity">center</item>
+        <item name="android:layout_marginEnd">@dimen/text_view_item_recycler_view_margin_end</item>
+    </style>
+
+    <style name="ImageViewItemRecyclerViewStyle" parent="TextAppearance.AppCompat.Display1">
```

Comment:
a style with only one property does not make sense, please put the property in the view directly

Reply:
done